scss/components/wysiwyg/_wysiwyg.scss

```diff
@@ -104,4 +115,32 @@
     clear: both;
   }
 
+//Special styles for Barb to use
+  h3.teal,
+  h2.teal {
+    color: #017c92;
+}
```

Comment:
These closing braces should be on the same indent as their opening tag. (2 spaces)

Reply:
Also fixing color while in there...